### PR TITLE
Issue #1083 D-pad, zoom, and cursor drag now working when graph view restricted to viewport

### PR DESCRIPTION
**Pull request by ceciliazaragoza**
D-pad, zoom, and cursor drag now enabled and limited to viewport when restrict to viewport checked.

Pull Request Checklist
- [x] Travis C.I. build passes
- [x] All five Demos look as expected when loaded
- [x] Reload works as expected
- [x] Loading file with Error brings up error modal, and it looks as expected
- [x] Loading file with Warning brings up warnings modal
- [x] Loading a network from the database looks as expected when loaded
- [x] Import works for both SIF and GraphML
- [x] Graph can be exported to SIF, GraphML and Excel
- [ ] Graph can be exported to PNG, SVG and PDF
- [x] Expression data and Additional Sheets can be exported to Excel
- [x] Print works as expected
- [x] Restrict graph to viewport works as expected (check/uncheck)
- [x] Viewport Size Changing works as expected (small/medium/large/fit)
- [x] Toggle between Grid Layout and Force Graph Layout works as expected
- [x] Force Graph Parameter Sliders change the number above them and have an effect on the graph
- [x] Locking/Unlocking/Resetting/Undo Resetting the force graph parameters works as expected
- [x] Enabling and disabling node coloring works as expected
- [x] Node coloring options work as expected (selection top/bottom dataset, averaging values, changing max value)
- [x] Weighted graph loaded with the "Enable Edge Coloring" option checked appears as expected
- [x] Hide/Show Weights works as expected (always/never/upon mouseover)
- [x] Edge Weight Normalization Factor can be changed (set/reset)
- [x] Gray Edge Threshold can be changed, slider changes the number and has an effect on the graph
- [x] Checking Show Gray Edges as Dashed works as expected (check/uncheck)
- [x] D-pad left/right/top/bottom/center works as expected
- [x] Zoom slider changes number and has effect on graph (viewport and menu)
- [x] Right click on a node opens gene page, page is populated with correct data

### CODE REVIEW

**Comment by ceciliazaragoza** [comment]
- [ ] Graph can be exported to PNG, SVG and PDF
- The D-pad, zoom slider, and cursor drag do not work as expected after exporting to SVG, but this issue is present in the master branch. More details can be found in: #1093.

**Comment by ceciliazaragoza** [comment]
- [ ] D-pad left/right/top/bottom/center works as expected
- D-pad doesn't go all the way to the edges, will try to fix in move function.

**Comment by ceciliazaragoza** [comment]
Now the D-pad can go to the edges of the graph.

File: web-client/public/js/graph.js [hunk 1]
```diff
@@ -359,20 +437,14 @@ export var drawGraph = function (workbook) {
     var restrictGraphToViewport = function (fixed) {
         if (!fixed) {
             $("#restrict-graph-to-viewport span").removeClass("glyphicon-ok");
-            $(document).ready(function () {
-                $(".scale-and-scroll").show();
-            });
             $("input[name=viewport]").removeProp("checked");
-            $container.addClass("cursorGrabbing");
+            // $container.addClass("cursorGrabbing");
```
Comment (dondi) [inline web-client/public/js/graph.js:441]: Can we delete this one outright since it’s commented out?

**Review by dondi — APPROVED** [review]
LGTM—one commented-out line to delete, but that can be done after merge

**Pull request merged**